packages/table/src/spectrum-table-cell.css

```diff
@@ -0,0 +1,209 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/* THIS FILE IS MACHINE GENERATED. DO NOT EDIT */
+:host([align='center']) {
+    text-align: center; /* .spectrum-Table-cell--alignCenter */
+}
+:host([dir='ltr'][align='end']) {
+    text-align: right; /* [dir=ltr] .spectrum-Table-cell--alignRight */
+}
+:host([dir='rtl'][align='end']) {
+    text-align: left; /* [dir=rtl] .spectrum-Table-cell--alignRight */
+}
+:host([dir='ltr']) {
+    padding-left: var(
+        --spectrum-table-regular-cell-padding-left
+    ); /* [dir=ltr] .spectrum-Table-cell */
+    padding-right: var(--spectrum-table-regular-cell-padding-right);
+}
+:host([dir='rtl']) {
+    padding-left: var(--spectrum-table-regular-cell-padding-right);
+    padding-right: var(
+        --spectrum-table-regular-cell-padding-left
+    ); /* [dir=rtl] .spectrum-Table-cell */
+}
+:host {
+    box-sizing: border-box; /* .spectrum-Table-cell */
+    font-size: var(--spectrum-table-regular-cell-text-size);
+    font-weight: var(--spectrum-table-regular-cell-text-font-weight);
+    line-height: var(--spectrum-table-regular-cell-text-line-height);
+    min-height: calc(
+        var(--spectrum-table-regular-cell-min-height) -
+            var(--spectrum-table-regular-cell-padding-top) -
+            var(--spectrum-table-regular-cell-padding-bottom)
+    );
+    padding-bottom: var(--spectrum-table-regular-cell-padding-bottom);
+    padding-top: var(--spectrum-table-regular-cell-padding-top);
+}
+:host {
+    position: relative; /* .spectrum-Table-cell,
+   * .spectrum-Table-headCell */
+}
+:host(.focus-visible),
+:host(.is-focused) {
+    outline: none; /* .spectrum-Table-cell.focus-ring,
+   * .spectrum-Table-cell.is-focused,
+   * .spectrum-Table-headCell.focus-ring,
+   * .spectrum-Table-headCell.is-focused */
+}
+:host(.is-focused),
+:host(:focus-visible) {
+    outline: none; /* .spectrum-Table-cell.focus-ring,
+   * .spectrum-Table-cell.is-focused,
+   * .spectrum-Table-headCell.focus-ring,
+   * .spectrum-Table-headCell.is-focused */
+}
+:host([dir='ltr']) .spectrum-Table-cell.is-focused:before,
```

Comment:
This points at something we may have missed in the styles. `.spectrum-Table-css` should be `:host` and `.is-focused` should likely be `[focused]`...